pkg/entity/batch_exporter/cmd.go

```diff
@@ -16,10 +17,13 @@ const (
 	includeExistingEventsFlag = "include-existing-events"
 )
 
+var content, _ = ioutil.ReadFile("pkg/entity/batch_exporter/docstring.md")
```

Comment:
Same question here. Does the CLI still work? And I'm unsure of the overhead here... @jankeesvanandel WDYT?

Reply:
I'm not in favor of using separate files here. A lot can go wrong when reading files and it makes startup performance worse (I think it will always read all files no matter the command you run).
I prefer to add a longDoc variable here, but not read it from file. Go supports multiline strings to make it more readable: https://www.golangprograms.com/how-do-you-write-multi-line-strings-in-go.html